### DISCUSSION TITLE
Option to apply styles not only to classes, but also to elements.

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#4979] [Ideas]
I would like a way to style the default style for all P, H1 to H6 tags.

There should still be a way to apply styles to a specific < P > element without affecting all others. (webflow does this same thing pretty clearly)

This styles should also be overriden if an element has a Class that affects those properties.

**Comment by lexoyo** [comment]
That's a great idea

Would it be a special syntax in the class selector or a different text box?

The ideal for my use case would be to be able to style things inside the element too, e.g. `.aclass p`
